src/Tribe/iCal.php

```diff
@@ -84,8 +84,15 @@ public function get_ical_link( $type = 'home' ) {
 	 */
 	public function month_view_ical_link( $event_date = null ) {
 		$tec = Tribe__Events__Main::instance();
+		$url = $tec->getLink( 'month', $event_date );
 
-		return add_query_arg( [ 'ical' => 1 ], $tec->getLink( 'month', $event_date ) );
+		/*
+		 * If the url contains "now" for the date, replace it with the current year-month.
+		 * Make sure we don't replace if it is part of another word - check for slashes.
+		 */
+		$url = str_replace( '/now/', '/' . gmdate( 'Y-m' ) . '/', $url );
```

Comment:
Do not use any system-based date handling (`date`, `gmdate`); use `Tribe__Date__Utils::build_date_object` instead.

Reply:
I would have a more precise targeting of the `now` string:

1. There could not be a trailing `/`.
2. "now" is the English version of this, what about localized strings?